compiler/plugins/input/StableHLO/Conversion/Preprocessing/test/complex_lowering.mlir

```diff
@@ -15,21 +15,6 @@ func.func @add(%arg0 : tensor<2xf32>, %arg1 : tensor<2xf32>, %arg2 : tensor<2xf3
   func.return %5, %6 : tensor<2xf32>, tensor<2xf32>
 }
 
-// CHECK-LABEL: @add_unranked
-func.func @add_unranked(%arg0 : tensor<*xf32>, %arg1 : tensor<*xf32>, %arg2 : tensor<*xf32>, %arg3 : tensor<*xf32>) -> (tensor<*xf32>, tensor<*xf32>) {
```

Comment:
Can you add a note to the PR description about why these unranked tests are removed? Maybe link to a relevant commit / email from stablehlo if unranked is no longer supported (I don't think IREE handles unranked anyways)

Reply:
There are unranked tests in multiple places. Do you think its good to add there? I was thinking of adding it in the commit log. Something like this:

```

Major change in this checkin:

1. Unranked dynamism was removed from StableHLO - http://go/stablehlo-dynamism-rfc#-p5-represent-shape-computations-as-stablehlo-operations-on-variadic-0-dimensional-tensors-and-drop-support-for-unranked-dynamism
```

Reply:
You can include it in both the PR description and the commit log.

We have GitHub configured to use the PR description as the commit message when merging by default: 
![image](https://github.com/openxla/iree/assets/4010439/9433dbae-a091-4f26-884f-4bac4f5d8c55)

(you can change the commit message if you'd like, and I do sometimes, but keeping information in both places is useful)

Reply:
Oh, and use a public link for that, `http://go/` only works for Google :P

Reply:
whoops 🤦🏽